index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <header>
+        <img src="./images/ic_menu.svg" alt="Menu" title="Menu">
+    </header>
```

Comment:
- [ ] The hamburger button on the right should include in a `nav` tag. :writing_hand:

Reply:
Hi @mahbubul14! 

Do I need to add a "Getting started section" if, at this moment, the project doesn't need any particular prerequisites, set up, install, usage, tests, or deployment?

Reply:
Hello @anagudelogu,
I can see your project is already approved. So, you don't need to add anything more. 
You can check [this link](https://github.com/mahbubul14/house_rent) to get an idea for the near future. Those parts you haven't worked on like "test", you don't need to add. You can just add a basic idea of how people can run the project.